### PR TITLE
DEATH TO LOSERS

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
FUCK IT
KILL LOSERS

who needs to test this shit amiright we use the losers for that
no runtimes here

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
don't close this yet we're gonna make the code even **BETTER**

File: hippiestation/code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -8,3 +8,9 @@
 	for(var/T in get_traumas())
 		var/datum/brain_trauma/BT = T
 		BT.on_life()
+
+/mob/living/carbon/Life()
+	. = ..()
+	if(rand(50))
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/carbon/life.dm:14]: if(client)

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (lcass) [reply hippiestation/code/modules/mob/living/carbon/life.dm:14]: If Life() is called every tick then this is just gonna clock up the chat, no.

Comment (Manly-Dwarf) [reply hippiestation/code/modules/mob/living/carbon/life.dm:14]: omega

**Comment by Eagle-Eyess** [comment]
idk this doesnt seem like a good pr...

**Comment by GuyonBroadway** [comment]
@Eagle-Eyess you lost your rights to complain when you lost the match

File: hippiestation/code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/mob/living/carbon/life.dm:14]: It will kill the **FAILURES** once and then be over and done with.

**Comment by Manly-Dwarf** [comment]
Got a loser to test it, and guess what, it runtimed every tick!
Fixed that.

File: hippiestation/code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (Manly-Dwarf) [reply hippiestation/code/modules/mob/living/carbon/life.dm:14]: okay fixed

**Review by GuyonBroadway — APPROVED** [review]

**Comment by geylard** [comment]
+1

**Comment by GuyonBroadway** [comment]
CODE IS GOOD!!

**Comment by Eagle-Eyess** [comment]
-1 we all winners at heart...

**Comment by frozen-is-a-fag** [comment]
+1 LOSER YOUR A LOSER ARE YOU FEELING SORRY FOR YOURSELF? WELL YOU SHOULD BECAUSE YOU'RE A LOSER

**Comment by CitrusGender** [comment]
image

**Comment by Carbonhell** [comment]
No

**Comment by Manly-Dwarf** [comment]
TYRANT
aaaaaaaaaaa

**Comment by blobbernaut** [comment]
pass it or i will return to fulfill the prophecy

**Comment by ghost** [comment]
what the fuck blob no
you're not allowed back
snail down nigga wtf

**Pull request closed without merging**